charts/values.yaml

```diff
@@ -0,0 +1,58 @@
+# -- Number of replicas for the deployment
+replicaCount: 1
+# -- YHS server address
+yhsServerAddr: ":8989"
+
+common:
+  # -- Labels to add to all deployed resources
+  labels: {}
+  # -- Namespace to deploy the resources
+  namespace: "yunikorn"
+
+image:
+  # -- Docker registry
+  registry: "docker.io"
+  # -- Docker image repository
+  repository: "gresearch/yunikorn-history-server"
+  # -- Docker image tag
+  tag: "latest"
+  # -- Docker image pull policy
+  pullPolicy: "IfNotPresent"
+
+service:
+  # -- Service type
+  type: LoadBalancer
```

Comment:
Can you change the default to `ClusterIP`?

Reply:
updated